README.md

```diff
@@ -8,13 +8,13 @@
 
 If you have access to a Mac (or a Linux laptop), you'll find this process much easier. But if you must use Windows, you'll find plenty of help online, so don't despair. Windows, especially Windows 10, has made working with Node much easier.
 
-That said, our demo will performed with a Mac.
+That said, our demo will be performed with a Mac.
 
-**Very important note**: If you diverge from these instructions in any way and get stuck, we _may_ not be able to help you. There are too many things that could go wrong. So please, the first time through, follow instructions to the letter as much as possible. On a second or third time through (and we recommend you try this several times, digging deeper each time until it all comes clear to you), you can start experimenting. But first time thorugh, _by the book, please_.
+**Very important note**: If you diverge from these instructions in any way and get stuck, we _may_ not be able to help you. There are too many things that could go wrong. So please, the first time through, follow instructions to the letter as much as possible. On a second or third time through (and we recommend you try this several times, digging deeper each time until it all becomes clear to you), you can start experimenting. But first time through, _by the book, please_.
```

Comment:
There is nothing wrong with "comes clear", however, as it's less common, I'll go with "becomes".